### PR TITLE
Add American football (sic) items.

### PR DESCRIPTION
**Pull request by ShiftyRail**
Courtesy of Bomber Harris.
Here are the images : 

image

image

NT one is in the vendors. Don't know where to put the syndie ones yet.

:cl:
- rscadd: The long-standing tradition of American football has finally reached the outermost areas of space. Get a suit representing Nanotrasen and practice for UniverseBowl today.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Oh great an icon conflict I love those

**Comment by ghost** [comment]
john madden john madden john madden john madden john madden

**Comment by Kurfursten** [comment]
Please add as a possible outfit for cryo men

**Comment by angryturnip** [comment]
These better give some kind of bonus for tackling

**Comment by Hinaichigo** [comment]
Footballs and holodeck football field when.

**Comment by PrimeDSS13** [comment]
top ten juiced up coders AND the top ten juiced up PRs

**Comment by ConfusedCactus** [comment]
god FINALLY i can run the assistant football team tackling everything in their path to the end zone gimmick (the end zone is the armory)

**Comment by ConfusedCactus** [comment]
put the syndie ones as emag-required contraband in the cargo console maybe?
you could also gate them behind just using the soldering iron on the cargo console board like the russian clothing crates are

**Comment by cloutcode** [comment]
americ anfootball items when

**Comment by Kurfursten** [comment]
> put the syndie ones as emag-required contraband in the cargo console maybe? you could also gate them behind just using the soldering iron on the cargo console board like the russian clothing crates are

Or hell, a large wooden crate that contains 1 wooden mannequin wearing the Syndie outfit. Call it a "Football tackling dummy" crate. It's normal for teams to put the colors of enemy teams on their tackling dummies. Similarly, it would be fun if the tackle_act() for mannequins would make them shake.

**Comment by Eneocho** [comment]
>no regular football (soccer)

do amerifats really

**Comment by Eneocho** [comment]
>NT one is in the vendors. Don't know where to put the syndie ones yet.

Put them on the vendor under contraband, like the Team Security Merch vendor, which has the team Geometer (cult) under the hacked tab. Would work for consistency as well.

**Comment by ManopolisJones** [comment]
football is great, but when are we getting GRAV BALL?

**Comment by Theophrastus1** [comment]
www.nfl.com

**Comment by kernikov** [comment]
https://www.youtube.com/watch?v=KxNGMvNIvP8

**Comment by ShiftyRail** [comment]
Wanted to do this this week-end but had a home flood

**Comment by ShiftyRail** [comment]
There it is. Thank you for your patience.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -848,6 +848,21 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
 	selection_from = list(Blue, Navy, Tan)
 
+/datum/supply_packs/nfl_suits
+	name = "NFL Suit and training dummy"
+	contains = list(/obj/structure/mannequin,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/clothing/suit/syndie_football,
+					/obj/clothing/suit/syndie_football,
```
Comment (Eneocho) [inline code/datums/supplypacks.dm]: Why. Just add the syndie football clothing to the vendor's contraband tab, so you keep consistent with the team security vendor, which has team geometer (cult) gear under contraband.

Comment (ShiftyRail) [reply code/datums/supplypacks.dm]: Its thr training dummy so you give it to train on the dummy

**Comment by Eneocho** [comment]
Man this shit is all over the place.
The suit on the suitlord, the hat on the autodrobe under premium and the syndie gear under a fucking supply crate for some goddamned reason.
Should've put the hat on the hatlord or the suit on the autodrobe as well. 
Also no point on keeping the helmet on premium, a magicbox with both items would've been better.

**Pull request merged**